db/migrate/20210603065037_create_github_tasks.rb

```diff
@@ -0,0 +1,21 @@
+class CreateGithubTasks < ActiveRecord::Migration[6.1]
+  def change
+    create_table :github_tasks do |t|
+      t.string :title, null: false
+      t.string :repo, null: false
+      t.string :issue_url, null: false, index: { unique: true }
+      t.string :opened_by_username, null: true
+      t.datetime :opened_at, null: false
+
+      t.column :action, :tinyint, null: true, default: 0
+      t.column :knowledge, :tinyint, null: true, default: 0
+      t.column :area, :tinyint, null: true, default: 0
+      t.column :size, :tinyint, null: true, default: 0
+      t.column :type, :tinyint, null: true, default: 0
```

Comment:
We _might_ want to add an index to speed this up, although there won't be that many records so the benefit might not that great. But I can look at the queries later

Reply:
Sure :)